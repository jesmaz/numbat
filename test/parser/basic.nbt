a + b
a - b
a * b
a / b
a << b
a >> b
a = b
var a
val b