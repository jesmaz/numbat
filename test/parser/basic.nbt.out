(a + b)
(a - b)
(a * b)
(a / b)
(a < < b)
(a > > b)
(a = b)
(var a)
(val b)
((var a) : c)